entrypoint.sh

```diff
@@ -16,6 +16,11 @@ if [[ -z "$SKIP_LABELS" ]]; then
   SKIP_LABELS="work-in-progress,wip"
 fi
 
+if [[ -z "$REMOVE_LABEL" ]]; then
+  echo "Set the REMOVE_LABEL env variable."
+  exit 1
```

Comment:
1 small Question: 
- Why we are making `REMOVE_LABEL` a mandatory parameter? 
- i think, this will break the existing workflows, right? 

so small request: can we make this param optional? or set default value "" if we want to make it a mandatory param? what do you say?

Reply:
Okay